tests/test_samples.py

```diff
@@ -0,0 +1,54 @@
+"""Snapshot test of formatting of sample files.
+
+Code samples are located in tests/samples. To add a new sample create a new
+file and create a snapshot of the formatting result with
+
+    $ pytest --snapshot-update
+
+Commit both the sample as well as the updated snapshot.
+"""
+
+import io
+from pathlib import Path
+
+import pytest
+from syrupy.assertion import SnapshotAssertion
+
+from testutils import zeekscript
+
+SAMPLES_DIR = Path("tests/samples")
```

Comment:
```suggestion
SAMPLES_DIR =  Path(__file__).parent / "samples" 
```

EDIT: Maybe or maybe not, as that will need mangling of `name` in to the call of `snapshot()`, though that might be okay.

```
+    name = str(sample.relative_to(SAMPLES_DIR.parent.parent))
     output = _format(input_)
     assert output == snapshot(
-        name=str(sample)
+        name=name
     ), f"formatted {sample} inconsistent with snapshot"

```